liveclasses/models.py

```
@@ -0,0 +1,64 @@
+                    .with_api_key(settings.JITSI_SECRET) \
+                    .with_user_name(self.teacher.username) \
+                    .with_user_email(self.teacher.email) \
+                    .with_moderator(True) \
+                    .with_app_id(settings.JITSI_APP_ID) \
+                    .with_user_avatar("") \
+                    .sign_with(reader.read())
+                return token
+        except Exception as e:
+            print(e)
+
+    def get_meeting_url_teacher(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_meeting_url_teacher`

Add return type annotation: `str`

(ANN201)